mr/src/main/java/org/apache/iceberg/mr/InputFormatConfig.java

```diff
@@ -93,6 +94,11 @@ public ConfigBuilder schema(Schema schema) {
       return this;
     }
 
+    public ConfigBuilder select(String[] columns) {
```

Comment:
Could we use a `List` instead of an array of strings here? It's usually easier if Iceberg converts to an array instead of expecting the caller to.

Reply:
I believe that this is using `String[]` as the `IcebergInputFormat` has been changed in this PR to call `org.apache.hadoop.conf.Configuration#getStrings` method which returns `String[]`.

However, I agree with the usage of collections (or Lists) vs `String[]`. It should be noted that `org.apache.hadoop.conf.Configuration` also has a `getStringCollection` function that could be called instead. 

Documentation on `getStringCollection`:
https://hadoop.apache.org/docs/current/api/org/apache/hadoop/conf/Configuration.html#getStringCollection-java.lang.String-